src/htrun/README.md

```diff
@@ -481,18 +481,15 @@ int main() {
     // 2. Send test case related data
     GREENTEA_SETUP(15, "gimme_auto");  // __timeout, __host_test_name
 
-    // ...
     // Send to master {{gimme_something; some_stuff}}
     greentea_send_kv("gimme_something", "some_stuff");
 
     char key[16] = {0};
     char value[32] = {0};
     // Blocking wait for master response for {{gimme_something; some_stuff}}
     greentea_parse_kv(key, value, sizeof(key), sizeof(value));
-    // ...
+    
     fprintf(stderr, "Received from master %s, %s", key, value);
-    // ...
-
     GREENTEA_TESTSUITE_RESULT(true);    // __exit
 }
 ```
```

Comment:
Snippets could potentially be converted into examples instead in the future.

Reply:
Yes once we created the example will update the Readme accordingly